src/main/java/com/stripe/model/PaymentIntent.java

```diff
@@ -335,4 +355,20 @@ public void setDestinationObject(Account c) {
     }
     // </editor-fold>
   }
+
+  @Getter
+  @Setter
+  @EqualsAndHashCode(callSuper = false)
+  public static class NextAction extends StripeObject {
+    NextActionRedirectToUrl redirectToUrl;
+    String type;
```

Comment:
I think we need the additional field? At least it shows up in my autogen PR https://github.com/stripe/stripe-java/pull/676/files.. Also confirm that it's present in public OpenAPI
```
    Map<String, Object> useStripeSdk;
```

Reply:
@mickjermsurawong-stripe This is undocumented/secret right now though so not something we'd want to ship for now.

Reply:
Confirmed, we want the parameter/enum value but this property should **not** be in the libraries since it's opaque and specific to stripe.js